app/routes/members-area.my-account/route.tsx

```diff
@@ -1,22 +1,193 @@
+                    <span className="paragraph1">View and edit your personal info below.</span>
+                </div>
+                <div className={styles.actions}>
+                    <button className="button secondaryButton smallButton" onClick={onDiscardClick}>
+                        Discard
+                    </button>
+                    <button
+                        className="button primaryButton smallButton"
+                        disabled={isUpdating}
+                        onClick={handleUpdate}
+                    >
+                        {isUpdating ? <Spinner size={22} /> : 'Update Info'}
+                    </button>
+                </div>
+            </div>
+
+            <div className={styles.section}>
+                <div>
+                    <h2 className="heading5">Personal info</h2>
+                    <span className="paragraph1">Update you personal information.</span>
+                </div>
+
+                <form className={styles.form}>
+                    <label>
+                        First Name
+                        <input
+                            className="textInput"
+                            value={firstName}
+                            onChange={(e) => setFirstName(e.target.value)}
+                        />
+                    </label>
+
+                    <label>
+                        Last Name
+                        <input
+                            className="textInput"
+                            value={lastName}
+                            onChange={(e) => setLastName(e.target.value)}
+                        />
+                    </label>
+
+                    <label>
+                        Phone
+                        <input
+                            className="textInput"
+                            value={phone}
+                            onChange={(e) => setPhone(e.target.value)}
+                        />
+                    </label>
+                </form>
+
+                <div className={styles.actions}>
+                    <button className="button secondaryButton smallButton" onClick={onDiscardClick}>
+                        Discard
+                    </button>
+                    <button
+                        className="button primaryButton smallButton"
+                        disabled={isUpdating}
+                        onClick={handleUpdate}
+                    >
+                        {isUpdating ? <Spinner size={22} /> : 'Update Info'}
+                    </button>
+                </div>
+            </div>
+
+            <div className={styles.section}>
+                <div>
+                    <h2 className="heading5">Login info</h2>
+                    <span className="paragraph1">View your login email and reset password.</span>
+                </div>
+
+                <div className={styles.loginInfoSection}>
+                    <div>
+                        <span>Login email:</span>
```

Comment:
use divs here and this css snippet becomes redundant
```
& > div {
        display: flex;
        flex-direction: column;
}
```

Reply:
done